src/components/NurseryWithdrawals/PlantingProgressMap.tsx

```diff
@@ -112,7 +112,7 @@ export default function PlantingProgressMap({ plantingSiteId, reloadTracking }:
 
   const updatePlantingComplete = useCallback(
     (id: number, val: boolean) => {
-      if (!selectedZoneHasStats) {
+      if (val === true || !selectedZoneHasStats) {
```

Comment:
I think that in practice this should not happen—there won't be statistics if the subzones aren't already all completed. I don't object to having this check in there though.

Reply:
Ah yes, that is true. Hmm.. I can remove it since it doesn't make sense.